lxl-web/src/lib/components/find/SearchCard.svelte

```diff
@@ -191,11 +221,9 @@
 				{/if}
 			{/key}
 		{/if}
-		{#if item.heldByMyLibraries?.length}
-			<div class="card-libraries flex items-start">
-				<MyLibsHoldingIndicator libraries={item.heldByMyLibraries} />
-			</div>
-		{/if}
+		<div class="hidden md:inline">
+			{@render holdingsButton()}
```

Comment:
Also, I feel the layout could be achieved without rendering the button twice. 

Using 
```
grid-template-areas : 
'image content debug'
'. libraries libraries'
```
For smaller screens, for example. Also, it should use a container query so it wraps when resizing the panels, not only the window.

Reply:
But I also wonder if the `libraries` section should maybe a column of the search card `footer` rather than the entire search card? The current layout takes up width of the entire card that could be used for long titles in narrow cards, for example.

<img width="447" alt="Skärmavbild 2025-06-27 kl  13 41 18" src="https://github.com/user-attachments/assets/ec83bbd6-ad5f-4f57-b046-fb2babcdd7b2" />